gcovr/tests/cmake_oos_ninja/CMakeLists.txt

```diff
@@ -1,7 +1,3 @@
-
 cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
 
-project(gcovrtest)
-
-    add_executable(gcovrtest main.cpp)
-
+include(../cmake_oos/CMakeLists.txt)
```

Comment:
Is this kind of coupling between tests sensible?

* pro: less duplication, easier maintenance
* con: might accidentally break this test in the future

Reply:
The intention was to reduce the copying of test data and to use a little bit complexer cmake files for testing.